platform/darwin/src/MGLShapeSource.h

```diff
@@ -143,6 +142,41 @@ extern const MGLShapeSourceOption MGLShapeSourceOptionSimplificationTolerance;
  */
 - (instancetype)initWithIdentifier:(NSString *)identifier shape:(nullable MGLShape *)shape options:(nullable NS_DICTIONARY_OF(MGLShapeSourceOption, id) *)options NS_DESIGNATED_INITIALIZER;
 
+/**
+ Returns a shape source with an identifier, a shape that is a 
+ `MGLShapeCollectionFeature` holding the features passed in, and dictionary of 
```

Comment:
It does _take_ an array but that is not what it returns in the "shape" property of the instantiated object. Isn't that what this sentence is trying to explain?